From: Gregoire GUTTIN <gregoire.guttin@apitech.fr>
Date: TODO
Subject: extend password length from 256 to 4096
See: https://github.com/cyrusimap/cyrus-sasl/pull/611
---
diff -ru a/saslauthd/lak.h b/saslauthd/lak.h
--- a/saslauthd/lak.h
+++ b/saslauthd/lak.h
@@ -62,9 +62,9 @@
 #define LAK_GROUP_MATCH_METHOD_ATTR 0
 #define LAK_GROUP_MATCH_METHOD_FILTER 1
 
-#define LAK_BUF_LEN 128
-#define LAK_DN_LEN 512
-#define LAK_PATH_LEN 1024
+#define LAK_BUF_LEN 4096
+#define LAK_DN_LEN 4096
+#define LAK_PATH_LEN 4096
 #define LAK_URL_LEN LAK_PATH_LEN
 
 typedef struct lak_conf {
diff -ru a/saslauthd/saslauthd-main.h b/saslauthd/saslauthd-main.h
--- a/saslauthd/saslauthd-main.h
+++ b/saslauthd/saslauthd-main.h
@@ -82,7 +82,7 @@
 #define DOOR_FILE		"/mux"              
 
 /* login, pw, service, realm buffer size */
-#define MAX_REQ_LEN		256     
+#define MAX_REQ_LEN		4096     
 
 /* socket backlog when supported */
 #define SOCKET_BACKLOG  	32
